trusted-committer-article.md

```diff
@@ -0,0 +1,232 @@
+contrasting the TC role to other roles in IS and why we think the name is both
+apt and important.
+
+- Contributor
+    - makes contributions
+    - sometimes part of the community
+    - sometimes sent by another part of the organization to develop a feature
+      for it. This is why contributors are sometimes referred to as Guests. 
+    - responsible for "fitting in"/conforming to the host team and it's 
+      expectations and processes
+    - similar to an Apache Developer or committer
+
+- Trusted Committer
+    - Sometimes referred to as a host.
+    - In this analogy, he is both resonsible for building the house and setting
+      the house rules to make sure his guests are comfortable.
+    - In comparison to the Contributor, a TC has rights to push code closer to
+      production
+- Product Owner
+    - Responsible for defining and prioritizing requirements and stories for
+      the community to implement.
+    - Also interacts directly and frequently with the trusted committers
+    - Distinction between the guest and host PO - have to negotiate
+      requirements, if necessary.
+    - Depending on the type of InnerSource community, there might not be a PO
+      from the outset. Especially true in case of innovation oriented and
+      bottom-up driven communities. POs come into play when a productization
+      of an InnerSource communities products becomes relevant.
+
+    ? In the Guest/Host analogy, what would be the analogon of a PO?
+
+## Why role names matter
+
+- The role of the TC is present in every successful InnerSource community but
+  not every community uses that name.
+- True for myself - I used the term Maintainer. It turns out that conficts with
+  the technical role "Maintainer" defined by GitHub, for instance. 
+- Apache e. g. also uses the term Committer, but attaches fewer and mostly
+  tech oriented responsibilities to that role. The TC role with its additional
+  community oriented responsibilities goes beyond that.
+- Similar to how naming is important in writing software, it is, too, for the
+  roles in an InnerSource project so that everyone has the same notion of 
+  the role.
+- The "Trusted" in TC conveys that she is trusted by both her community and
+  empowered by her management to do the job.
+
+## Ensuring Product Quality
+
+- define quality goals 
+- set contribution guidelines 
+- enact/communicate these guidelines 
+- own tech related decisions 
+- act as stakeholder of the SW/Architecture towards rest of org
+- set an example
+- make sure community has what it needs to produce quality sw
+    - communicated quality goals/standards
+    - empowerment by management
+    - right tools
+    - prioritize quality and release dates
+- ensure quality at strategic level (longevity of the SW)
+- ensure that decisions regarding quality are followed through
+- review and accept contributions
+- document yet undocumented requirements
```

Comment:
I might flip the action on this one - the TC should ensure that when an undocumented requirement (or process, or limitation) is discovered that they are responsible for making sure it is documented accurately. The beginner's perspective on the discovery is probably more effectively captured by the contributor, but the TC has to validate that there isn't a general principle or a misunderstanding in the contributor's document. So the TC is responsible for the content, but they also need to make sure that it is written such that the newcomer understands it without needing additional coaching.

Reply:
Excellent point, @cewilliams !